components/org.wso2.carbon.data.provider/src/main/java/org/wso2/carbon/data/provider/siddhi/config/SiddhiDataProviderConfig.java

```diff
@@ -18,20 +18,19 @@
 
 package org.wso2.carbon.data.provider.siddhi.config;
 
-import com.google.gson.Gson;
 import com.google.gson.JsonElement;
-import com.google.gson.JsonObject;
 import com.google.gson.JsonParser;
 import org.wso2.carbon.data.provider.ProviderConfig;
 
-import java.util.List;
-
 public class SiddhiDataProviderConfig implements ProviderConfig {
 
     private String siddhiApp;
     private JsonElement queryData;
     private int publishingInterval;
     private String timeColumns;
+    private int currentPage;
+    private int pageSize;
+    private boolean paginationEnable;
```

Comment:
rename the variable as isPaginationEnabled.

Reply:
Fixed in dd7e38e